lingua_franca/internal.py

```diff
@@ -460,12 +460,19 @@ def _call_localized_function(func, *args, **kwargs):
 
             # Check if we need to add timezone awareness to any datetime object
             if config.inject_timezones:
-                for k, v in kwargs.items():
-                    if isinstance(v, datetime) and v.tzinfo is None:
-                        kwargs[k] = to_local(v)
-                for idx, v in enumerate(args):
-                    if isinstance(v, datetime) and v.tzinfo is None:
-                        args = args[:idx] + (to_local(v),) + args[idx + 1:]
+                for name_of_the_keyword_argument, \
+                    value_of_the_keyword_argument in kwargs.items():
+                    if isinstance(value_of_the_keyword_argument, datetime) \
+                            and value_of_the_keyword_argument.tzinfo is None:
+                        kwargs[name_of_the_keyword_argument] = to_local(
+                            value_of_the_keyword_argument)
+                for index_of_the_argument, value_of_the_argument in \
+                        enumerate(args):
+                    if isinstance(value_of_the_argument, datetime) \
+                            and value_of_the_argument.tzinfo is None:
+                        args = args[:index_of_the_argument] + \
+                               (to_local(value_of_the_argument),) + \
+                               args[index_of_the_argument + 1:]
```

Comment:
🤡

Reply:
Apparently there's no clown face, so I guess it's emoji math 😆 + 😒 🧀 + 🤖 = 🏅

Reply:
I understand that there are different opinions on the use of single character variables particularly in contained loops however I don't think it is helpful to turn the PR into a joke. This seems to be the only reason for this PR not being merged now is that right?

Reply:
the joke is how this PR was handled

if this was blocking the PR getting merged noone made any comment about it

if code style is blocking this PR , then i remind that this change was explicitly requested in opposition to what the repo maintainers (ex-maintainer in my case) wanted, so don't come and tell me the PR is blocked because of this change.... this PR is not merged because the maintainers are not allowed to maintain the repo